Teams.Security/Teams.Security.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Please fix that. As I mention class libraries should not target .net core, you should remember this as a golden rule when you  are implementing .net core application.